public/components/overview/mitre/components/techniques/techniques.tsx

```diff
@@ -177,29 +181,47 @@ export const Techniques = withWindowSize(class Techniques extends Component {
     }
   }
 
+  async getMitreTechniques () {
+    const data = await WzRequest.apiReq('GET', '/mitre/techniques', {}).then(res => res).catch(err => mitreTechniques);
+    const result = (((data || {}).data || {}).data || {}).affected_items;
```

Comment:
it will throw if data comes from the catch 
`.catch(err => mitreTechniques);`
I think there is needed a better error handling